source/common/router/router.h

```diff
@@ -331,6 +331,9 @@ class Filter : Logger::Loggable<Logger::Id::router>,
   // Http::StreamFilterBase
   void onDestroy() override;
 
+  absl::optional<std::string> getShadowCluster(const ShadowPolicy& shadow_policy,
```

Comment:
This should probably be private (I assume that anonymous namespace won't work here).

Reply:
Done